### ISSUE TITLE
Improve Documentation about Flame Graph (with --sample-time examples)

### ISSUE THREAD

**Issue by vimkim** [namhyung/uftrace#1932]
## Improve Documentation about Flame Graph

It seems like the flame graph is skipping a few functions if their durations are below a certain threshold. I was trying to generate a graph as dense as possible, minimizing the number of omitted functions.

Maybe due to my lack of foundational knowledge, reading the documentation about `uftrace dump` was not enough to figure out how to create such a dense flame graph.

I saw somewhere that I can use:

```
uftrace dump --sample-time 1us --flame-graph <...>
```

but I am unsure of other options.

#### My Questions Are:

1. Can I set the sample time below 1 microsecond? Maybe to 0.0001 microseconds?
2. Are there any options that guarantee the most dense version of the flame graph, preferably similar to that of Chrome tracing (if I may be so greedy)?

Thank you. If someone can provide information on this, I'm willing to create a PR to improve the documentation, which I hope will help the project.

**Comment by namhyung** [comment]
Yep, you can use `1ns` for nano-second precision.  Please feel free to send a PR for update!
